### ISSUE TITLE
synchronize SID values with latest draft-ietf-anima-constrained-voucher

### ISSUE THREAD

**Issue by mcr** [AnimaGUS-minerva/voucher#1]
https://github.com/AnimaGUS-minerva/voucher/blob/5da549f0fa307c75c8c8b11d4ae0a8ee2966c383/src/sid_data.rs#L97
has ancient SID allocations.  The new ones are at:

https://www.ietf.org/archive/id/draft-ietf-anima-constrained-voucher-15.html#name-sid-values-2
and
https://www.ietf.org/archive/id/draft-ietf-anima-constrained-voucher-15.html#name-sid-values

**Comment by mcr** [comment]
It looks like this example is correct:

~~~~
obiwan-[projects/trentonio/voucher](2.6.6) mcr 10087 %cbor2diag.rb data/00-D0-E5-F2-00-02/voucher_00-D0-E5-F2-00-02.vch 
18([h'A10126', {}, h'A1190993A501666C6F6767656402C11A5F56D1770B7130302D44302D45352D46322D30302D3032077658537946344C4C49697155322D4F476B366C46434167087902744D49494230544343415661674177494241674942416A414B42676771686B6A4F50515144417A42784D5249774541594B435A496D695A50794C47514247525943593245784754415842676F4A6B69614A6B2F49735A41455A46676C7A5957356B5A577874595734785144412B42674E5642414D4D4E794D3855336C7A64475674566D467961574669624755364D4867774D4441774D4441774E4759354D5446684D4434675657357A64484A31626D6367526D3931626E526861573467513045774868634E4D5463784D5441334D6A4D304E5449345768634E4D546B784D5441334D6A4D304E544934576A42444D5249774541594B435A496D695A50794C47514247525943593245784754415842676F4A6B69614A6B2F49735A41455A46676C7A5957356B5A57787459573478456A415142674E5642414D4D43577876593246736147397A6444425A4D424D4742797147534D34394167454743437147534D34394177454841304941424A5A6C5548493075702F6C33655A6639764342622B6C496E6F454D45676337526F2B585A43746A414930434431664A664A522F68497979446D48577959694E46625243483966796172666B7A67583470307A54697A716A4454414C4D416B4741315564457751434D414177436759494B6F5A497A6A304541774D44615141775A674978414C514D4E75726638747635306C524F443544515848454F4A4A4E5733515632673951456444536B324D592B416F537242536D47534E6A68346F6C454F6845754C674978414A346E57664E772B426A625A6D4B6949695545635477484D68475658614D48592F46376E333977774B634242534F6E644E507143704F454C6C36627133435A71513D3D', h'63CC823A34B964ADC835B58E2EE1E7E30088ADE6896F94B13AC730643E9660B5A93453F3C9D8A09AB57A0113A406727884763A2AD04B4FAB4F6FB8BCB32EFA47'])

obiwan-[projects/trentonio/voucher](2.6.6) mcr 10089 %echo A1190993A501666C6F6767656402C11A5F56D1770B7130302D44302D45352D46322D30302D3032077658537946344C4C49697155322D4F476B366C46434167087902744D49494230544343415661674177494241674942416A414B42676771686B6A4F50515144417A42784D5249774541594B435A496D695A50794C47514247525943593245784754415842676F4A6B69614A6B2F49735A41455A46676C7A5957356B5A577874595734785144412B42674E5642414D4D4E794D3855336C7A64475674566D467961574669624755364D4867774D4441774D4441774E4759354D5446684D4434675657357A64484A31626D6367526D3931626E526861573467513045774868634E4D5463784D5441334D6A4D304E5449345768634E4D546B784D5441334D6A4D304E544934576A42444D5249774541594B435A496D695A50794C47514247525943593245784754415842676F4A6B69614A6B2F49735A41455A46676C7A5957356B5A57787459573478456A415142674E5642414D4D43577876593246736147397A6444425A4D424D4742797147534D34394167454743437147534D34394177454841304941424A5A6C5548493075702F6C33655A6639764342622B6C496E6F454D45676337526F2B585A43746A414930434431664A664A522F68497979446D48577959694E46625243483966796172666B7A67583470307A54697A716A4454414C4D416B4741315564457751434D414177436759494B6F5A497A6A304541774D44615141775A674978414C514D4E75726638747635306C524F443544515848454F4A4A4E5733515632673951456444536B324D592B416F537242536D47534E6A68346F6C454F6845754C674978414A346E57664E772B426A625A6D4B6949695545635477484D68475658614D48592F46376E333977774B634242534F6E644E507143704F454C6C36627133435A71513D3D | pretty2diag.rb 
{2451: {1: "logged", 2: 1(1599525239), 11: "00-D0-E5-F2-00-02", 7: "XSyF4LLIiqU2-OGk6lFCAg", 8: "MIIB0TCCAVagAwIBAgIBAjAKBggqhkjOPQQDAzBxMRIwEAYKCZImiZPyLGQBGRYCY2ExGTAXBgoJkiaJk/IsZAEZFglzYW5kZWxtYW4xQDA+BgNVBAMMNyM8U3lzdGVtVmFyaWFibGU6MHgwMDAwMDAwNGY5MTFhMD4gVW5zdHJ1bmcgRm91bnRhaW4gQ0EwHhcNMTcxMTA3MjM0NTI4WhcNMTkxMTA3MjM0NTI4WjBDMRIwEAYKCZImiZPyLGQBGRYCY2ExGTAXBgoJkiaJk/IsZAEZFglzYW5kZWxtYW4xEjAQBgNVBAMMCWxvY2FsaG9zdDBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABJZlUHI0up/l3eZf9vCBb+lInoEMEgc7Ro+XZCtjAI0CD1fJfJR/hIyyDmHWyYiNFbRCH9fyarfkzgX4p0zTizqjDTALMAkGA1UdEwQCMAAwCgYIKoZIzj0EAwMDaQAwZgIxALQMNurf8tv50lROD5DQXHEOJJNW3QV2g9QEdDSk2MY+AoSrBSmGSNjh4olEOhEuLgIxAJ4nWfNw+BjbZmKiIiUEcTwHMhGVXaMHY/F7n39wwKcBBSOndNPqCpOELl6bq3CZqQ=="}}
~~~~

**Comment by j-devel** [comment]
Regarding the SID values in https://www.ietf.org/archive/id/draft-ietf-anima-constrained-voucher-15.html#name-sid-values

```
     2513 data .../proximity-registrar-pubk
     2512 data .../proximity-registrar-pubk-sha256
```
Any reason why 2513 comes before 2512?  (is it typo or ...)

**Comment by mcr** [comment]
Hi, the numbers just comes out of whatever pyang allocated when it processed the YANG modules.
I think that it gets output in alphabetical order, and ".../proximity-registrar-pubk" sorts before ".../proximity-registrar-pubk-sha256"

**Comment by j-devel** [comment]
Addressed in 7e743b3 and a3d35ed, thanks!
